styles/buttons.css

```diff
@@ -0,0 +1,27 @@
+.button {
+  border-radius: 0.6rem;
+  padding: 1.2rem 2.4rem;
+  max-width: 24rem;
+  min-width: 22rem;
+  text-align: center;
+}
+
+.buttonPrimary {
+  background-color: var(--dodger-blue);
+  color: var(--black-pearl);
+}
+
+.buttonPrimary:hover {
+  background: var(--eucalyptus);
+  background: linear-gradient(to right, var(--dodger-blue), var(--eucalyptus));
+}
+
+.buttonTertiary {
```

Comment:
why tertiary? we don't have secondary

Reply:
I still haven t thounght about the design of the `buttonSecondary`